contracts/PolygonTokenBridger.sol

```diff
@@ -59,15 +72,20 @@ contract PolygonTokenBridger is Lockable {
     /**
      * @notice Constructs Token Bridger contract.
      * @param _destination Where to send tokens to for this network.
-     * @param _l1Weth Ethereum WETH address.
+     * @param _polygonRegistry L1 registry that stores updated addresses of polygon contracts.
```

Comment:
Can you add a comment explaining how to set this on L2 since its described as an "L1 registry"

Reply:
Sure. Can do. It should still be the L1 address to ensure that this gets deployed at an identical address on both chains.

Reply:
Done.